R/descriptives.R

```diff
@@ -382,7 +382,7 @@ DescriptivesInternal <- function(jaspResults, dataset, options) {
   makeSplit <- splitName != ""
   dataset <- .readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = if (makeSplit) splitName)
   dataset.factors <- jaspBase::readDataSetByVariableTypes(options, "variables")
-  if (makeSplit)
+  if (makeSplit && length(variables) > 0)
     dataset.factors[[splitName]] <- dataset[[splitName]]
```

Comment:
but this causes `dataset.factors[[splitName]]` to not exist in this particular scenario. I'm not sure if that will cause errors down the road. What about
```suggestion
  if (makeSplit)
    if (length(variables) > 0)
      dataset.factors[[splitName]] <- dataset[[splitName]]
    else
      dataset.factors <- dataset
```